.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/tinyusb"]
 	path = lib/tinyusb
 	url = https://github.com/hathach/tinyusb
+[submodule "lib/pico-sdk"]
+	path = lib/pico-sdk
+	url = https://github.com/raspberrypi/pico-sdk.git
```

Comment:
Could we use the `PICO_SDK_FETCH_FROM_GIT=1` CMake variable instead of adding the pico-sdk as a submodule?

Reply:
The makefile now checks if `PICO_SDK_PATH` is defined and acts accordingly.
https://github.com/greatscottgadgets/apollo/pull/34/commits/e1f5a04bd7033b949b4217533119a6e4ee0485b7#diff-43e5a6ff56e05cff146ed6f928508217da39b0db828c180d7de01e57b42d0790R12